etc/helm/pachyderm/templates/pachd/deployment.yaml

```diff
@@ -67,6 +67,10 @@ spec:
           value: "/pach"
         - name: ETCD_PREFIX
           #value:
+        {{- if .Values.jaeger.enabled }}
+        - name: JAEGER_ENDPOINT
+          value: jaeger-collector:14268
```

Comment:
IIRC, pachd shouldn't need this environment variable (i.e. this is a pachctl/client-side thing). I believe pachd uses `JAEGER_COLLECTOR_SERVICE_HOST` which is set by Kubernetes after the Jaeger service is created?

Reply:
ah yes that makes sense